scipy/stats/morestats.py

```diff
@@ -32,11 +33,20 @@ def __dir__():
 def __getattr__(name):
     if name not in __all__:
         raise AttributeError(
-            "scipy.stats.morestats is deprecated and has no attribute "
-            f"{name}. Try looking in scipy.stats instead.")
+            "`scipy.stats.morestats` is deprecated and has no attribute "
+            f"`{name}`.")
```

Comment:
```suggestion
            "`scipy.stats.morestats` is has no attribute `{name}`; furthermore, "
            "it is deprecated and will be removed in SciPy 2.0.0.")
```

Reply:
"it" could be clarify to say we are talking about the namespace and not the attribute.